tests/integration_test.go

```diff
@@ -1,20 +1,27 @@
 package integration_test
 
 import (
-	"context"
+	"bytes"
+	"io"
 	"io/ioutil"
+	"log"
 	"os"
 	"os/exec"
+	"path/filepath"
 	"testing"
-
-	"github.com/numtide/nar-serve/go-nix/libstore"
 )
 
 func TestHappyPath(t *testing.T) {
+	// _ := assert.New(t)
+
 	accessKeyID := "Q3AM3UQ867SPQQA43P2F"
 	secretAccessKey := "zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG"
+	// os.Setenv("MINIO_ACCESS_KEY", accessKeyID)
+	// os.Setenv("MINIO_SECRET_KEY", secretAccessKey)
+	// os.Setenv("AWS_ACCESS_KEY_ID", accessKeyID)
+	// os.Setenv("AWS_SECRET_ACCESS_KEY", secretAccessKey)
 
-	dataDir, err := ioutil.TempDir("", "nar")
+	dataDir, err := ioutil.TempDir(".", "nar")
```

Comment:
don't re-use the same folder, it defeats the purpose of the tempdir

Reply:
Yeah, I put it there so I can see it in my vscode editor. Once all fixed, I will remove it.